docker-compose.server.override.yml

```diff
@@ -9,3 +9,18 @@ services:
   nginx:
     volumes:
       - ./freescout:/app
+
+  supervisor:
+    image: itkdev/supervisor-php8.1:alpine
```

Comment:
Can't it run in php 8.3.... support for php 8.1 (Ended 7 months ago (25 Nov 2023))...

Reply:
Changed